### ISSUE TITLE
False positive for missing-f-string-syntax (RUF027)

### ISSUE THREAD

**Issue by ashrub-holvi** [astral-sh/ruff#21939]
### Summary

Hi,

code:
```
def func():
    label = ""
    nice_label = "{label} xxx"
    return nice_label.format(label=label)
```
triggers `missing-f-string-syntax` (RUF027)
see in [playground](https://play.ruff.rs/64961571-bdcd-4266-94bf-307dcd7b7c49)
but I guess it can be just f-string, so, perhaps `UP032` can be triggered instead

**Comment by ashrub-holvi** [comment]
but if `nice_label` used more as a template more than one times, it correct to use .`format`

**Comment by ntBre** [comment]
Thanks for the report! It looks like this is a known problem with the rule with a draft PR trying to improve the situation (https://github.com/astral-sh/ruff/pull/13076), but no tracking issue, until now :)
